bosh-director/lib/bosh/director/deployment_plan/stemcell.rb

```diff
@@ -79,6 +90,32 @@ def spec
         }
       end
 
+      def cid_for_az(az)
+        raise 'please bind model first' if @models.nil?
```

Comment:
I kept is a generic runtime exception on purpose - it should always happen during develop time and bubble up to the developer (it means he didn't bind the model before using `cid_for_az`). 

What's the preferred way in such situations? Should we still introduce a more specific error? Does it need to be put in `errors.rb` (even though it should never leave the system)?